vernac/metasyntax.ml

```diff
@@ -1467,7 +1468,7 @@ let recover_squash_syntax sy =
 let make_pa_rule (typs,symbols) parsing_data =
```

Comment:
@herbelin this doesnt' build on CI, maybe
```suggestion
let make_pa_rule level (typs,symbols) parsing_data =
```
and add `sd.level` to the call in `make_parsing_rules`?

Reply:
Indeed, the rebase was broken. The level is now in `parsing_data`, so let's take it from there. Sorry. Thanks for telling.